static/app/utils/useTeams.tsx

```diff
@@ -172,8 +228,28 @@ function useTeams({limit}: Options = {}) {
     }
   }
 
+  useEffect(() => {
+    if (slugs) {
+      const teamsInStore = store.teams.filter(t => slugs.includes(t.slug));
+
+      // Only fetch the slugs if they do not exist in the store already
+      if (teamsInStore.length !== slugs.length) {
+        loadTeamsBySlug();
+      }
+    } else if (provideUserTeams && !store.loadedUserTeams) {
```

Comment:
This is just me, but I find `else if` kind of hard to read. How do you feel about this


```tsx
useEffect(() => {
	  // Load specified team slubs
	  if (slugs) {
		    const teamsInStore = store.teams.filter(t => slugs.includes(t.slug));
		    
		    // Only fetch the slugs if they do not exist in the store already
		    if (teamsInStore.length !== slugs.length) {
			    loadTeamsBySlug();
		    }
		    return
	  }

	  // Load user teams
	  if (provideUserTeams && !store.loadedUserTeams) {
	  		loadUserTeams();
	  }
}, ...)

Reply:
Also I wonder if we should represent in the types that you can only pass slugs OR `provideUserTeams`

Reply:
Yup i can make these changes. In the future is this format with 
```
if () {
    ...
    return;
}
if () {
    ...
    return;
}
```

preferred to `if/else if`?